### PR TITLE
Click on reminder title in list to view individual reminder item

### LINKED ISSUE #3: See An Individual Reminder
When the user clicks on one of the reminders on the page, they are taken to an individual reminders route (ie: ‘/reminders/1’, if the id of the reminder is 1). The title of the reminder object should be displayed.

So, what needs to happen here?

* You’ll need a reminder route in addition to the reminders route.
* It should be nested inside the reminders route.
* The user should still see all of the reminders from your previous feature. The specific reminder they are looking at now should be rendered in the outlet of the reminders.hbs template.
* There should be some CSS for the .active class.

There is a failing test set up in `tests/acceptance/reminder-list-test.js` to get you started - this test needs to be rewritten to reflect the route you are expecting to visit.

### PR DESCRIPTION
**Pull request by esayler**
Resolves #3 
@martensonbj @Tman22 

## Purpose

Add ability to click on one of the reminders, which takes the user to a view of an individual reminder based on a route based on the ID of the idea.

## Approach

Add in a nested route to view an individual reminder by clicking on a title of a specific reminder within the list of reminder items. Show a specifc reminder item's data above the list when hitting that route.

#### Ember.js Guides
* [Ember.js Guides - Triggering changes with actions](https://guides.emberjs.com/v2.11.0/components/triggering-changes-with-actions/)
* [ember-cli-sass](https://www.npmjs.com/package/ember-cli-sass)
* [Ember.js Guides - Routing/Redirection](http://guides.emberjs.com/v2.11.0/routing/redirection/)
* [Ember.js Guides - Templates/Links](https://guides.emberjs.com/v2.11.0/templates/links/)

#### Git stuff
* [Atlassian Git Tutorial - Formatting Log Output](https://www.atlassian.com/git/tutorials/git-log#formatting-log-output)
* [git docs - `git cherry-pick`](https://git-scm.com/docs/git-cherry-pick)

### Open Questions and Pre-Merge TODOs

- [x] When the user clicks on one of the reminders on the page, they are taken to an individual reminders route (ie: ‘/reminders/1’, if the id of the reminder is 1). The title of the reminder object should be displayed.
- [x] reminder route nested inside reminders route.
- [x] The user should still see all of the reminders from your previous feature. The specific reminder they are looking at now should be rendered in the outlet of the reminders.hbs template.
- [x] There should be some CSS for the .active class.
- [x] There is a failing test set up in tests/acceptance/reminder-list-test.js to get you started - this test needs to be rewritten to reflect the route you are expecting to visit.

### Test coverage

Fixed acceptance test on `reminder-list` to ensure that clicking on a specific `reminder-item` (title) link will route to `/reminders/:reminder_id` and render a `reminder` template that displays the reminder's appropriate data from Mirage. 

### Follow-up tasks

- [Issue #5 - Create a New Note](https://github.com/turingschool-projects/1610-remember-3/issues/5)

#### Before

screen capture on 2017-02-08 at 17-29-47 1

#### After

screen capture on 2017-02-09 at 16-05-22

### CODE REVIEW

File: tests/unit/routes/index-test.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import { moduleFor, test } from 'ember-qunit';
```
Comment (martensonbj) [inline tests/unit/routes/index-test.js]: Don't need this file

File: tests/unit/routes/reminders/reminder-test.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { moduleFor, test } from 'ember-qunit';
```
Comment (martensonbj) [inline tests/unit/routes/reminders/reminder-test.js]: Don't need this file either

File: mirage/factories/reminder.js [hunk 3]
```diff
@@ -1,8 +1,8 @@
 import { Factory, faker } from 'ember-cli-mirage';
 
 export default Factory.extend({
-  title: () => faker.lorem.words().capitalize(),
+  title: () => faker.company.bs().capitalize(),
   date: () => faker.date.recent(3),
-  notes: () => faker.lorem.paragraph(),
+  notes: () => faker.hacker.phrase(),
```
Comment (martensonbj) [inline mirage/factories/reminder.js:6]: Great idea.

**Review by martensonbj — CHANGES_REQUESTED** [review]
Couple extra files to remove

**Comment by esayler** [comment]
@martensonbj removed the empty tests!

**Comment by martensonbj** [comment]
:shipit: 🐬 :gorilla:

**Pull request merged**